### ISSUE TITLE
ACDD 1.3

### ISSUE THREAD

**Issue by lukecampbell** [ioos/compliance-checker#127]
Update compliance checker to support ACDD 1.3
http://wiki.esipfed.org/index.php/Attribute_Convention_for_Data_Discovery_1-3

**Comment by dpsnowden** [comment]
Please ensure that all implemented versions of ACDD are supported.  Do not migrate away from the ACDD 1.2, keep both.  i.e. 

```bash
> compliance-checker input.nc --acdd-1.2
> compliance-checker input.nc --acdd-1.3 
```
both work.

**Comment by benjwadams** [comment]
I have a branch I was working on for this some time ago here: 
https://github.com/benjwadams/compliance-checker/tree/acdd_1.3
Still WIP and does not allow for swapping of versions currently, but probably better than starting from scratch.

**Comment by benjwadams** [comment]
Should be implemented now that #262 is merged. Closing.
